ampbench_lib.js

```diff
@@ -775,10 +775,14 @@ function lib_fetch_cdn_validator_spec_file_revision(callback) {
  */
 function lib_load_validator(opt_force_reload) {
     let _force_reload = opt_force_reload || false;
-    const _validator = amphtml_validator.getInstanceFromFileSync(VALIDATOR_JS_FILE, _force_reload);
-    amphtml_validator_instance = _validator; // cache the module level instance here
-    amphtml_validator_spec_file_revision = lib_extract_validator_spec_file_revision(_validator.scriptContents);
```

Comment:
Thank you. I've merged this into my version and made another update to use sha256 and rename this into signature consistently. Haven't tested it thus far (about to do this), will update this thread if I find some problem.